lib/AcqCheckboxFilter/AcqCheckboxFilter.js

```diff
@@ -21,6 +21,15 @@ const AcqCheckboxFilter = ({
   onChange,
   options = [],
 }) => {
+  const dataOptions = options.map(option => (
```

Comment:
actually, `dataOptions` could be much complex than that.
```
    dataOptions: PropTypes.arrayOf(PropTypes.shape({
      disabled: PropTypes.bool,
      label: PropTypes.node,
      readOnly: PropTypes.bool,
      value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
    })).isRequired,
```

Reply:
https://github.com/folio-org/stripes-acq-components/blob/master/lib/FieldSelect/FieldSelect.js#L18